packages/ember-glimmer/tests/integration/helpers/render-test.js

```diff
@@ -0,0 +1,272 @@
+import { observer } from 'ember-metal/mixin';
+import Controller from 'ember-runtime/controllers/controller';
+import { RenderingTest, moduleFor } from '../../utils/test-case';
+import { set } from 'ember-metal/property_set';
+import EmberRouter from 'ember-routing/system/router';
+
+
+moduleFor('@htmlbars Helpers test: {{render}}', class extends RenderingTest {
+  ['@test should render given template']() {
+    this.owner.register('controller:home', Controller.extend());
+    this.registerTemplate('home', '<p>BYE</p>');
+
+    this.render(`<h1>HI</h1>{{render 'home'}}`);
+
+    this.assertText('HIBYE');
+  }
+
+  ['@test should render nested helpers']() {
+    this.owner.register('controller:home', Controller.extend());
+    this.owner.register('controller:foo', Controller.extend());
+    this.owner.register('controller:bar', Controller.extend());
+    this.owner.register('controller:baz', Controller.extend());
+
+    this.registerTemplate('home', '<p>BYE</p>');
+    this.registerTemplate('foo', `<p>FOO</p>{{render 'bar'}}`);
+    this.registerTemplate('bar', `<p>BAR</p>{{render 'baz'}}`);
+    this.registerTemplate('baz', `<p>BAZ</p>`);
+
+    this.render('<h1>HI</h1>{{render \'foo\'}}');
+    this.assertText('HIFOOBARBAZ');
+  }
+
+  ['@test should have assertion if neither template nor view exists']() {
```

Comment:
Can you tweak this title and remove `view` (we don't support the view part anymore and I think I missed it when I updated te original test)?